tests/test_shell.py

```diff
@@ -86,3 +86,33 @@ def test_init_flowproject_template(self, template):
         self.call(f"python -m flow init -t {template}".split())
         assert str(signac.get_project()) == "project"
         assert os.path.exists("project.py")
+
+    def test_template_create_base(self):
+        self.call("python -m flow init".split())
+        self.call("python -m flow template create".split())
+        assert os.path.exists("templates/script.sh")
+        template = flow.environment.get_environment().template
```

Comment:
I think `get_environment` is exposed as a top-level function, so let's use that.

```suggestion
        template = flow.get_environment().template
```

Also I would prefer to force this test to use a provided environment rather than actually detecting the environment, to ensure that tests pass reliably from any system. You have to do this in a way that will affect both the current Python environment _and_ subprocesses (`python -m flow ...`). I recommend a monkeypatch that sets an environment variable. Search `SIGNAC_FLOW_ENVIRONMENT` in the tests for an example, I just used this same trick for PR #533.